test/integration/attachment_access_limited_integration_test.rb

```diff
@@ -162,6 +167,7 @@ def add_file_attachment(filename, to:)
     to.attachments << FactoryBot.build(
       :file_attachment,
       attachable: to,
+      title: filename,
```

Comment:
Should this change be in a different commit? It's currently in "Assert setup complete in AttachmentAccessLimitedIntegrationTest" but doesn't seem to fit with the other changes.

Reply:
I think it was needed by the `find('.existing-attachments a', text: 'greenpaper.pdf')` that I added, otherwise the title was set to the default for the `file_attachment` factory which was a bit confusing. I'll add something to the commit note to explain.

Reply:
It turns out it was needed in the "when attachment is replaced" scenario, because the original attachment title is used in the flash notice which I want to assert against. I've added a paragraph to the commit note to explain this.

Reply:
While I was at it, I've changed the `refute_nil` to `assert` which seems better.